5.6/test/run

```diff
@@ -162,10 +175,23 @@ test_application() {
 
   test_connection
   check_result $?
+
+  info "Checking the PHP_OPTIONS"
```

Comment:
you could docker run the image and then use docker cp to copy the config out of the container and inspect it.